lib/gcloud/dns/record/list.rb

```diff
@@ -45,6 +45,18 @@ def next
         end
 
         ##
+        # Retrieve all records and add to the current list.
+        # This changes the contained records.
```

Comment:
@quartzmo: Thoughts on a better documentation for this method? I'm stumped on what to say.